### PR TITLE
feat: Skip attendance for holidays from Upload Attendance

### PR DESCRIPTION
**Pull request by Anurag810**
Upload attendance does not show Holidays in the template.

**Please check once locally**

### CODE REVIEW

File: erpnext/hr/doctype/upload_attendance/upload_attendance.py [hunk 1]
```diff
@@ -63,16 +66,34 @@ def get_data(args):
...
 				employee.name, employee.employee_name, date,
 				existing_attendance and existing_attendance.status or "",
 				existing_attendance and existing_attendance.leave_type or "", employee.company,
 				existing_attendance and existing_attendance.naming_series or get_naming_series(),
 			]
+			if date in holidays[employee_holiday_list]:
+				row[4] =  "Holiday"
 			data.append(row)
+
+			print(data)
```
Comment (deepeshgarg007) [inline erpnext/hr/doctype/upload_attendance/upload_attendance.py]: Remove print

File: erpnext/hr/doctype/upload_attendance/upload_attendance.py [hunk 2]
```diff
@@ -123,6 +144,16 @@ def upload():
 	frappe.enqueue(import_attendances, rows=rows, now=True if len(rows) < 200 else False)
 
 def import_attendances(rows):
+
+	def remove_holidays(rows):
+		from pprint import pprint
+		pprint(rows)
```
Comment (deepeshgarg007) [inline erpnext/hr/doctype/upload_attendance/upload_attendance.py]: remove print

File: erpnext/hr/doctype/upload_attendance/upload_attendance.py [hunk 3]
```diff
@@ -123,6 +144,16 @@ def upload():
...
+		rows = [ row for row in rows if row[4] != "Holiday"]
+		for row in rows:
```
Comment (deepeshgarg007) [inline erpnext/hr/doctype/upload_attendance/upload_attendance.py]: remove print

File: erpnext/hr/utils.py [hunk 4]
```diff
@@ -400,7 +400,7 @@ def get_holidays_for_employee(employee, start_date, end_date):
 				"holiday_list": holiday_list,
 				"start_date": start_date,
 				"end_date": end_date
-			})
+			}, debug = 1)
```
Comment (deepeshgarg007) [inline erpnext/hr/utils.py]: remove debug

**Review by deepeshgarg007 — CHANGES_REQUESTED** [review]

**Comment by deepeshgarg007** [comment]
@Anurag810 Why refactor?

**Review by nabinhait — APPROVED** [review]

**Pull request merged**